hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -42,6 +42,12 @@
 @st.defines_strategy_with_reusable_values
 def from_dtype(dtype):
     # type: (np.dtype) -> st.SearchStrategy[Any]
+    """Creates a strategy which can generate any value of the given dtype."""
+    if not isinstance(dtype, np.dtype):
```

Comment:
You can replace this block with an existing validator.

```python
from hypothesis.internal.validators import check_type

check_type(np.dtype, type, 'dtype')
```

Reply:
```python
from hypothesis.internal.validation import check_type
check_type(np.dtype, dtype, 'dtype')
```

Reply:
potato potat-oh 😉

Reply:
:potato:, :sweet_potato: 

Anyway, I think I'm done here.  Any final review comments, or do you think it's ready to merge?